migrations/migrations.sql

```diff
@@ -5460,14 +5460,65 @@ do $$
         values (8002, 9001, 'v6.9.4', 'drydock', 'aarch32_u16', 'v6.9.4', 'aarch32_u16', 'aarch32_u16reqproc', false, true, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
       end if;
 
+      --------------------------------------------------------
+      -------------- v6.10.4 Templates ------------------------
+      --------------------------------------------------------
+
+      -- x86_64 Ubuntu_14.04
+      if not exists (select 1 from "runtimeTemplates" where "archTypeCode" = 8000 and "osTypeCode" = 9000 and "version" = 'v6.10.4') then
+        insert into "runtimeTemplates" ("archTypeCode", "osTypeCode", "version", "drydockOrg", "drydockFamily", "drydockTag", "defaultTaskImage", "reqProcImage", "isDefault", "createdAt", "updatedAt")
+        values (8000, 9000, 'v6.10.4', 'drydock', 'u16', 'v6.10.4', 'u14all', 'u14reqproc', false, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
```

Comment:
Shouldn't drydockFamily be `u14` here?

Reply:
u16 family has all u14 images and runtimeTemplates for previous versions also has the same u16 family.